etc/requirements.txt

```diff
@@ -40,4 +40,5 @@ stop-words ~= 2018.7
 djangorestframework ~= 3.9
 django-debug-toolbar ~= 1.11
 openpyxl ~= 2.6
-django-picklefield ~= 2.0
\ No newline at end of file
+django-picklefield ~= 2.0
+geocoder ~= 1.38.1
```

Comment:
This is strange. Here it's correct, but when running `bash configure.sh`, it seems that this line is changed to `geocoder1.38.1` instead of `geocoder ~= 1.38.1`. This makes the script fail.

I will investigate it further, but I accept help.

Reply:
The problem was with `pyproject.toml` file. Solved this :smiley: